milatools/cli/commands.py

```diff
@@ -801,3 +727,193 @@ def _forward(
     webbrowser.open(url)
     proc.local_port = port
     return proc
+
+
+def setup_ssh_config_interactive(
+    ssh_config_path: Union[str, Path] = "~/.ssh/config",
+    _input: Optional[PipeInput] = None,
+):
+    """Interactively sets up some useful entries in the ~/.ssh/config file on the local machine.
+
+    Exits if the User decides to not set one of these entries, or doesn't confirm the change when
+    prompted.
+
+    Entries:
+    - "mila": Used to connect to a login node.
+    - "mila-cpu": Used to connect to a compute node.
+    - "mila-gpu": Used to connect to a compute node with a GPU.
+
+    Other entries:
+    - "*.server.mila.quebec !*login.server.mila.quebec": Sets some useful attributes for connecting
+      directly to compute nodes.
+
+    TODO: Add an interactive prompt to add entries for the ComputeCanada/DRAC clusters.
+    """
+
+    cfgpath = Path(ssh_config_path).expanduser()
+    if not cfgpath.exists():
+        if not yn(f"There is no {ssh_config_path} file. Create one?", _input=_input):
+            exit("No ssh configuration file was found.")
+        sshpath = cfgpath.parent
+        if not sshpath.exists():
+            sshpath.mkdir(mode=0o700, exist_ok=True)
+        cfgpath.touch(mode=0o600)
+        print(f"Created {cfgpath}")
+
+    ssh_config = SSHConfig(cfgpath)
+
+    # Check for a mila entry in ssh config
+    # NOTE: If there is no `mila` entry, ssh_config.host("mila") returns an empty dictionary.
+    username = ssh_config.host("mila").get("user")
```

Comment:
This will not catch a `Host` that could look like
```
Host                            mila secondhostname [...]
        User                    user
```
even if `ssh mila` would match that existing configuration

Reply:
Fixed in e111649